update.go

```diff
@@ -219,6 +228,12 @@ func (b *UpdateBuilder) Where(pred interface{}, args ...interface{}) *UpdateBuil
 	return b
 }
 
+// From sets the FROM clause of the query.
+func (b *UpdateBuilder) From(table string) *UpdateBuilder {
```

Comment:
1. According [to the doc](https://www.postgresql.org/docs/9.6/static/sql-update.html) `from_list` can be a list, not a single table name. Would be great to have it implemented.
2. Could you please mention in the comment that this is PG-specific?